dbms/src/Storages/DeltaMerge/ColumnFile/ColumnFileSetWithVectorIndexInputStream.h

```diff
@@ -69,7 +69,7 @@ class ColumnFileSetWithVectorIndexInputStream : public VectorIndexBlockInputStre
         , data_provider(data_provider_)
         , ann_query_info(ann_query_info_)
         , valid_rows(std::move(valid_rows_))
-        , vec_index_cache(context_.global_context.getLocalIndexCache())
+        , vec_index_cache(context_.global_context.getHeavyLocalIndexCache())
```

Comment:
Add comment why the local index used in ColumnFile is "heavy" and that in DMFile is "light"?

Reply:
done